src/tests/sql/init_version_7.sql

```diff
@@ -0,0 +1,45 @@
+DROP TABLE `inventory`;
+
+CREATE TABLE IF NOT EXISTS `inventory` (
+  `hash` blob NOT NULL,
+  `objecttype` int DEFAULT NULL,
+  `streamnumber` int NOT NULL,
+  `payload` blob DEFAULT NULL,
+  `integer` integer NOT NULL,
+  UNIQUE(hash) ON CONFLICT REPLACE
+) ;
```

Comment:
I don't see a difference from the previous version - in the `init_version_6.sql`

Reply:
I haven't reviewed the SQL files yet, I'm focusing on getting the structure of classes and tests right, it's already at like 3rd iteration.

Reply:
I looked through the tests and instructed to redesign how they perform setup.